cloud/azure/storage/variables.tf

```
@@ -0,0 +1,2617 @@
+}
+
+variable "blobservices_requests_error_transformation_function" {
+  description = "Transformation function for blobservices_requests_error detector (mean, min, max)"
+  type        = string
+  default     = "max"
+}
+
+variable "blobservices_requests_error_transformation_window" {
+  description = "Transformation window for blobservices_requests_error detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "5m"
+}
+
+variable "blobservices_requests_error_threshold_critical" {
+  description = "Critical threshold for blobservices_requests_error detector"
+  type        = number
+  default     = 90
+}
+
+variable "blobservices_requests_error_threshold_warning" {
+  description = "Warning threshold for blobservices_requests_error detector"
+  type        = number
+  default     = 70
+}
+
+variable "blobservices_requests_error_aperiodic_duration" {
+  description = "Duration for the blobservices_requests_error block"
+  type        = string
+  default     = "10m"
+}
+
+variable "blobservices_requests_error_aperiodic_percentage" {
+  description = "Percentage for the blobservices_requests_error block"
+  type        = number
+  default     = 0.9
+}
+
+# Fileservices_requests_error detectors
+
+variable "fileservices_requests_error_disabled" {
+  description = "Disable all alerting rules for fileservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "fileservices_requests_error_disabled_critical" {
+  description = "Disable critical alerting rule for fileservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "fileservices_requests_error_disabled_warning" {
+  description = "Disable warning alerting rule for fileservices_requests_error detector"
+  type        = bool
+  default     = null
+}
+
+variable "fileservices_requests_error_notifications" {
+  description = "Notification recipients list for every alerting rules of fileservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "fileservices_requests_error_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of fileservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "fileservices_requests_error_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of fileservices_requests_error detector"
+  type        = list
+  default     = []
+}
+
+variable "fileservices_requests_error_aggregation_function" {
+  description = "Aggregation function and group by for fileservices_requests_error detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name'])"
+}
+
+variable "fileservices_requests_error_transformation_function" {
+  description = "Transformation function for fileservices_requests_error detector (mean, min, max)"
+  type        = string
+  default     = "max"
```

```suggestion
  default     = "min"
```
This transformation was inverted in previous implementation.